### PR TITLE
✨ (shorts-api): Fixed imports

### PR DESCRIPTION
**Pull request by BenjaminOsmers**
Fixed bugs that cause request-for-access api to fail.

### CODE REVIEW

**Review by KearonF — DISMISSED** [review]

**Comment by BenjaminOsmers** [comment]
@arneschreuder Can I pull from upstream develop, and push my branch again? Or should I wait?

File: .eslintrc.json [hunk 1]
```diff
@@ -18,6 +18,11 @@
               }
             ]
           }
+        ],
+        "no-empty-function": "off",
```
Comment (arneschreuder) [inline .eslintrc.json:22]: I don't think this is right. If you need to change linting rules to pass your linting, this is wrong. Please remove this and remove the empty function/constructor that prompted this.

**Review by arneschreuder — CHANGES_REQUESTED** [review]

**Comment by arneschreuder** [comment]
I think if you update your branch and resolve the conflicts you should be good to go. Check the comment I made above about `eslintrc.json`. Please remove that contribution.

File: .eslintrc.json (hunk 1 above)
Comment (BenjaminOsmers) [reply .eslintrc.json:22]: Will do. What caused this is my query service class, how can I resolve it? If you look under api/shorts/service/queries/impl, that file contains the empty constructor. How would I go about fixing it?

**Comment by arneschreuder** [comment]
I think I already fixed it in my PR. So just update your branch and remove estlintrc changes. :)

**Review by arneschreuder — APPROVED** [review]
LGTM

**Comment by arneschreuder** [comment]
Shipit!

**Pull request merged**